pfhedge/nn/modules/bs/european.py

```diff
@@ -108,7 +155,17 @@ def delta(
 
         Returns:
             torch.Tensor
+
+        Note:
+            args are not optional if it doesn't accept derivative in this initialization.
```

Comment:
```suggestion
            Parameters are not optional if the module has not accepted a derivative in its initialization.
```

nit: args->parameters (as it is in documentation webpage)